tools/install.sh

```diff
@@ -155,7 +156,15 @@ function sync_dependencies {
 		 [ "$LINUX_VERSION" == "Debian"  ] || \
 		 [ "$LINUX_VERSION" == "Kali"    ] || \
 		 [ "$LINUX_VERSION" == "Ubuntu"  ]; then
-		apt-get install -y libfreetype6-dev python3-dev python3-pip pkg-config
+		apt-get install -y libfreetype6-dev python3-dev pkg-config
+		if ! python3 -m pip --version; then
+			if apt-get install python3-pip; then
+				echo "INFO: Installed python3-pip via apt-get"
+			else
+				curl https://bootstrap.pypa.io/get-pip.py | python3
```

Comment:
piped into `python3`. But cannot really confirm but all documentation from pypi this is the site and method to use if it is not available via a package manager. 

https://pip.pypa.io/en/stable/installing/
https://github.com/pypa/get-pip#usage